src/main/java/org/jenkinsci/test/acceptance/server/JenkinsControllerPoolProcess.java

```diff
@@ -57,8 +58,8 @@
     @Option(name="-n",usage="Number of instances to pool. >=1.")
     public int n = Integer.getInteger("count",1);
 
-    @Option(name="-socket",usage="Unix domain socket file to communicate with client")
-    public File socket = SOCKET;
+    @Option(name="-socket",usage="Unix domain socket file to communicate with client") @Inject @Named("socket")
```

Comment:
Args4j injection is done before Guice injection. Is that what we want, envvar overriding commandline option?